docs/snippets/all/tutorials/send_columns_arrays.cpp

```diff
@@ -0,0 +1,53 @@
+// Use the `send_columns` API to send several point clouds over time in a single call.
+
+#include <array>
+#include <rerun.hpp>
+#include <vector>
+
+using namespace std::chrono_literals;
+
+int main() {
+    const auto rec = rerun::RecordingStream("rerun_example_send_columns_arrays");
+    rec.spawn().exit_on_failure();
+
+    // 5 point clouds each with 4 points.
+    std::vector<std::array<float, 3>> positions = {
+        // clang-format off
+        {1.0, 0.0, 1.0}, {0.5, 0.5, 2.0},
+        {1.5, -0.5, 1.5}, {1.0, 1.0, 2.5}, {-0.5, 1.5, 1.0}, {-1.5, 0.0, 2.0},
+        {2.0, 0.0, 2.0}, {1.5, -1.5, 3.0}, {0.0, -2.0, 2.5}, {1.0, -1.0, 3.5},
+        {-2.0, 0.0, 2.0}, {-1.5, 1.5, 3.0}, {-1.0, 1.0, 3.5},
+        {1.0, -1.0, 1.0}, {2.0, -2.0, 2.0}, {3.0, -1.0, 3.0}, {2.0, 0.0, 4.0},
+        // clang-format on
+    };
+
+    // At each time stamp, all point clouds share the same but changing color.
+    std::vector<uint32_t> colors = {0xFF0000FF, 0x00FF00FF, 0x0000FFFF, 0xFFFF00FF, 0x00FFFFFF};
+
+    // Log at seconds 10-14
+    auto times = rerun::Collection{10s, 11s, 12s, 13s, 14s};
+    auto time_column = rerun::TimeColumn::from_times("time", std::move(times));
+
+    // Interpret raw positions and color data as rerun components and partition them.
+    auto indicators = std::vector(5, rerun::Points3D::IndicatorComponent());
+    auto indicator_batch =
+        rerun::PartitionedComponentBatch::from_loggable(rerun::take_ownership(std::move(indicators))
+        );
+    auto position_batch = rerun::PartitionedComponentBatch::from_loggable(
+        rerun::Collection<rerun::components::Position3D>(std::move(positions)),
+        {2, 4, 4, 3, 4}
```

Comment:
We should note here somewhere that these are the lengths of each cloud in `positions`

Reply:
For instance, calling this `from_loggable_with_length`